### PR TITLE
Add Products entity(part_1)

### PR DESCRIPTION
**Pull request by oshtaier**
Going to add basic things in that PR and add more views in the next ones

### CODE REVIEW

File: airgun/views/product.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+class ProductsTableView(BaseLoggedInView, SearchableViewMixin):
+    title = Text("//h2[contains(., 'Products')]")
+    new = Text("//button[contains(@href, '/products/new')]")
+    edit = Text(
+        "//td/a[contains(@ui-sref, 'product.repositories') and "
+        "contains(@href, 'products')]"
+    )
+    repo_discovery = Text("//button[contains(.,'Repo Discovery')]")
+    actions = ActionsDropdown("//td//div[contains(@class, 'btn-group')]")
```
Comment (abalakh) [inline airgun/views/product.py]: Could you please point me where's that actions list inside table cell? I'm a bit confused, but that's how table view looks for me (and this locator doesn't work there for obvious reasons):
image

File: airgun/views/product.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    actions = ActionsDropdown("//div[contains(@class, 'btn-group')]")
+    dialog = ConfirmationDialog()
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(
+            self.return_to_all, exception=False) is not None
+
+    @View.nested
+    class Details(SatTab):
```
Comment (abalakh) [inline airgun/views/product.py]: nitpick: lowercase

File: airgun/views/product.py (hunk 1 above)
Comment (oshtaier) [reply airgun/views/product.py]: will rename class work for you?

Comment (oshtaier) [reply airgun/views/product.py]: because I was talking about 'select action' dropdown obviously

Comment (abalakh) [reply airgun/views/product.py]: You misunderstood me - key words were "inside table cell". Your locator points to some `//td`, but there's no table surrounding that actions dropdown. Wrong locator then?

**Pull request merged**